adafruit_azureiot/quote.py

```diff
@@ -11,14 +11,22 @@
 safe arg.
 
 """
+
+from __future__ import annotations
+
+try:
+    from typing import Any, Union
+except ImportError:
+    pass
+
 _ALWAYS_SAFE = frozenset(
     b"ABCDEFGHIJKLMNOPQRSTUVWXYZ" b"abcdefghijklmnopqrstuvwxyz" b"0123456789" b"_.-~"
 )
 _ALWAYS_SAFE_BYTES = bytes(_ALWAYS_SAFE)
-SAFE_QUOTERS = {}
+SAFE_QUOTERS: dict = {}
```

Comment:
We need to remove this one for now as well.

Reply:
Removed type annotation.